### PR TITLE
TC 33 and 34 are committed

### PR DESCRIPTION
**Pull request by henrybc08**
imagen

Pull request for TC 33 and 34

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/accident/UpdateAccidentForm.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    @FindBy(xpath = "//form[@id='form-accident']/input[@value='Update']")
+    private WebElement updateButton;
+
+    public UpdateAccidentForm() {
+        CommonEvents.isVisible(updateButton);
+    }
+
+    public void updateAccidentInformationForm() {
+
```
Comment (lizeth-salazar) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/accident/UpdateAccidentForm.java]: Remove this blank space please

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/ssia/SSIAStepDefinitions.java [hunk 2]
```diff
@@ -134,7 +136,10 @@ public void workItemListIsShowedInPage() throws Throwable {
     private SafetyMenu safetyMenu;
     private AccidentList accidentList;
```
Comment (lizeth-salazar) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/ssia/SSIAStepDefinitions.java]: Please remove blank space

**Review by lizeth-salazar — CHANGES_REQUESTED** [review]
Please remove blank spaces

**Comment by henrybc08** [comment]
captura de pantalla 2018-07-02 a las 11 04 14

summary: Observations were corrected and conflicts were solved and tested

**Pull request merged**